spyder/app/mainwindow.py

```diff
@@ -94,10 +94,15 @@
 
 #==============================================================================
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
-# be set before creating the application
+# be set before creating the application. 
 #==============================================================================
+from spyder.config.main import CONF
+if CONF.get('main', 'high_dpi_scaling'):
+    high_dpi_scaling=True
+else:
+    high_dpi_scaling=False

```

Comment:
@mariacamilaremolinagutierrez, please add spaces around equals, like this

    high_dpi_scaling = True